app/components/pages/ThankYou/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import gql from 'graphql-tag'
+import { Query } from 'react-apollo'
+import ThankYou from './ThankYou'
+
+const MANUSCRIPT_TITLE = gql`
+  query GetManuscript($id: ID!) {
+    manuscript(id: $id) {
+      meta {
+        title
+      }
+    }
+  }
+`
+const ThankYouPage = ({ match }) => (
+  <Query query={MANUSCRIPT_TITLE} variables={{ id: match.params.id }}>
+    {({ data, loading, error }) => {
+      if (loading) {
+        return <div>Loading...</div>
+      }
+
+      if (error) {
+        return <div>{error.message}</div>
```

Comment:
Perhaps we could use `ErrorPage` in here? @tamlyn @diversemix what do you think?

Reply:
This looks good with the `ErrorPage` component, I'll make this gave and re-commit.